src/components/VideoCard/VideoCard.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import "./videoCard.css"
+
+export const VideoCard = ({video}) => {
+    const {thumbnail,creator,title} = video
+    return(
+        <>
+          <div className="video-card margin-xs padding-xs">
+              <div className="img-container">
+                <img className="img-responsive" src={thumbnail} alt="video_thumbnail"/>
+              </div>
+              <div>
+                  <h4 className="video-title text-sm">{title}</h4>
+                  <p className="video-creator text-md">{creator}</p>
+              </div>
+          </div>
```

Comment:
use avatars along with creator name
keep the font size of title greater then creater